setup.py

```diff
@@ -2,34 +2,26 @@
 
 __version__ = "0.1.0"
 
-# More information on setting values:
-# https://github.com/Sceptre/project/wiki/sceptre-template-handler-template
-
-# lowercase, use `-` as separator.
-TEMPLATE_HANDLER_NAME = 'custom-template-handler'
-# the template_handler call in sceptre e.g. type: custom_template_handler.
-TEMPLATE_HANDLER_TYPE = 'custom_template_handler'
-# do not change. Rename template_handler/template_handler.py to template_handler/{TEMPLATE_HANDLER_TYPE}.py
-TEMPLATE_HANDLER_MODULE_NAME = 'template_handler.{}'.format(TEMPLATE_HANDLER_TYPE)
-# CamelCase name of template_handler class in template_handler.template_handler.
-TEMPLATE_HANDLER_CLASS = 'CustomTemplateHandler'
-# One line summary description
-TEMPLATE_HANDLER_DESCRIPTION = 'A Custom Template Handler'
-# if multiple use a single string with comma separated names.
-TEMPLATE_HANDLER_AUTHOR = 'Sceptre'
+TEMPLATE_HANDLER_NAME = 'sceptre-sam-handler'
+TEMPLATE_TYPE = 'sam'
+TEMPLATE_HANDLER_MODULE_NAME = 'sam_handler.handler.py'
+TEMPLATE_HANDLER_CLASS = 'SAM'
+TEMPLATE_HANDLER_DESCRIPTION = 'Packages and renders SAM templates for use'
+TEMPLATE_HANDLER_AUTHOR = 'Jon Falkenstein'
 # if multiple use single string with commas.
-TEMPLATE_HANDLER_AUTHOR_EMAIL = 'sceptre@sceptre.org'
```

Comment:
Does this really need to be on the setup? I don't particularly want to put my email address on github to be scraped by all the world's bots...

Reply:
Who does that email address go to? You?

Reply:
it goes into the ether 😄